cloud-openstack/src/main/java/com/sequenceiq/cloudbreak/cloud/openstack/common/OpenStackImageVerifier.java

```diff
@@ -37,6 +47,8 @@ public void exist(OSClient<?> osClient, String name) {
             throw new CloudConnectorException(String.format("OpenStack image: %s not found with ids: %s", name, String.join(", ", imageIds)));
```

Comment:
The exception should mentioned that multiple images found with the same name, not that the image not found, shouldn't it?